### DISCUSSION TITLE
Datacite

### DISCUSSION THREAD

**Discussion by NRayya** [NFDI4Chem/nmrxiv#948] [Data Schemas]
Datcite has been under discussion in several task areas in NFDI4Chem. Here we try to reach to a final conclusion, and we mention the decisions taken in the last dedicated call. We welcome your comments. And when a decision is already provided, would be great to confirm your agreement with the decision number. (e.g., 1.1, I agree)

1. Creator fields: Currently provided in nmrXiv as "Authors" field which is filled by the users
1.1  Terminology: Should we keep the field name as "Authors" or switch to Creators?
    - DataCite preserves the term "Creator" for datasets and "Author" for publications.
    - RADAR4Chem: "Creator/Author"
    - Chemotion: "Author"
    - **Decision: "Creators/Authors": To comply with datacite and also to make it clear that we expect the names of the authors of the article corresponding to the dataset. In addition, using the plural is preferred to supply multiple creators**.

    1.2 Roles for "Authors": Currently authors in nmrXiv can have roles assigned to them from the "contributorType" controlled list in datacite.
    - **No final decision was taken here. There were suggests to keep authors and contributors as separate fields.**

2. Contributor:  Currently provided in nmrXiv as roles assigned to the authors only.
  2.1 Should we dedicate a separate field for contributors?
    - DataCite: Contributors are a separate field holding simialr or different values from the authors
    - RADAR4Chem: separate field for contributor(see [DS1](https://www.radar-service.eu/radar/en/dataset/RADXoObNHVOoRFkX.Supplementary%2Bdata%2B%2522Medicago%2BTERPENE%2BSYNTHASE%2B10%2BIs%2BInvolved%2BDefense%2BAgainst%2BOomycete%2BRoot%2BPathogen%2522), [DS2](https://www.radar-service.eu/radar/en/dataset/xopPmSIfYQucznOU.Supplementary%2Bdata%2B%2522Nor-guanacastepene%2Bpigments%2BChilean%2Bmushroom%2BCortinarius%2Bpyromyxa%2522), [DS3](https://www.radar-service.eu/radar/en/dataset/SsvwghnJyPVnAdHx.Supplementary%2Bdata%2B%2522Synthesis%2Bbiological%2Bevaluation%2Bhighly%2Bpotent%2Bfungicidal%2Bdeoxy-Hygrophorones%2522))
    - Chemotion: There is a feature under development to add contributors.
    - nmrXiv: only authors and submitters can have additional role as contributors.
      **Decision: Have a separate field for contributors.**

    2.2 Should we automatically assign some contributors roles (submitters and shared with)?
    - RADAR4Chem: Nothing assigned automatically
    - Chemotion: submitters are contributors as a researcher, and they can become an author by checking a box.
    - nmrXiv: submitters are contributors as a "Other"  contributorType.
      **Decision:
          No automated assignment
          Keep the responsibility on the user
          Keep things clear.
          Enable easy access to add submitters and shared with as creators or contributors
          For the contributor types, limit the options to be related to chemistry**

    2.3 Is there a need to assign more than one contributor role (knowing that the whole contributor field is not even mandatory in Datacite)? **No decision** 
  
    2.4 What about team members? should they be added as Data curators ? (2.2 suggests not)
  
    2.5 Citation tab:
  <img width="1308" alt="image" src="https://github.com/NFDI4Chem/nmrxiv/assets/82588017/ec66afcf-1847-476a-b7b3-2a47bc574a6e">
  We only provide the autors here. Should we also provide contributors or data submitters?

**Comment by NRayya** [comment]
Relevant previous discussion: https://github.com/NFDI4Chem/nmrxiv/discussions/630

**Comment by tilfischer** [comment]
Dear Noura,

very much sorry for the very delayed response. I will go though all of your points:
- 1.1, I agree -> Creators/Authors
- 1.2, I agree to separate the list of Authors/Creators and Contributors, but why shall we use roles for Authors/Contributors, if this is not part of DataCite schema? I would omit this, as I do not see the use case.

FYI: I recently uploaded something to Zenodo. To my surprise, Zenodo asked for a role of a Creator, while the Creator does not have such a role in DataCite. Possibly, they provide a role in their own (internal) Schema but do not send this to DataCite. However, they also do not even show this on the respective record page.

When submitting:

![Bildschirmfoto 2024-01-05 um 10 16 28](https://github.com/NFDI4Chem/nmrxiv/assets/85482437/a9e34278-aa83-4fce-bec8-ec7f51f9d793)

Published:
![Bildschirmfoto 2024-01-05 um 10 29 06](https://github.com/NFDI4Chem/nmrxiv/assets/85482437/0cc380a7-c67f-4323-97d3-73e072843994)
The role, which was also given for the Contributors is shown, while the role of the Creators isn't shown.

- 2.1, I agree -> separate fields for Contributors
- 2.2: I agree with your decisions and suggestions in bold face. I also would not suggest to automatically add this. Researcher could be a default role, but just on the first glance. If contributors are researchers which had a substantial contribution to the work, then they must be Authors(/Creators) following the guidelines of good scientific practice. The most probable role of a contributor with data(set) publishing from my point of view is DataCurator. This should not be automatically added, but maybe suggested first. 
- 2.3: I would suggest to only allow one role, simply for the reason that DataCite schema 4.4 gives for the occurrences of contributorType ``1 = required, but not repeatable``
- 2.4: This is connected to #630 . As written there, I would leave the power to the people but guide them. The team members should not be automatically added, but they could be shown as suggestions to the users to easily add them by picking someone from the list of suggested team members. 
- 2.5: This is an interesting point. I would append them to the list of Authors/Creators and put their roles in parentheses. Contributors have contributed, hence, they should be displayed. 

Best,
Tillmann

**Comment by tilfischer** [comment]
Dear all,

another test dataset got published https://nmrxiv.org/P55 or https://doi.org/10.57992/nmrxiv.p55 . The project shows two submitters with me being the first, although I didn't even started the submission but this was done by the other submitter. 

I want to highlight again that this should be changed to separate field for Authors/Creators and Contributors and nmrXiv Team members should NOT be automatically added.

Best,
Tillmann
